Project management/SEO preparation.md

```diff
@@ -0,0 +1,163 @@
+SEO stands for search engine optimization. It's a way of optimizing data on your website so that the search engines can recognize it and drive organic traffic to it.
+
+Even though we are using [Yoast SEO](https://wordpress.org/plugins/wordpress-seo/) plugin to handle most of the SEO-relevant metrics (from content analysis to meta tags, and structured data), there are some things that we need to make sure are set on the website before releasing it live.
+
+We have two scenarios we'll have to cover:
+
+1. a new website without the prior existence on the internet
+2. an existing website that we need to migrate and regain or improve their existing search ranking.
+
+### New website
+
+When we are creating a new website, we need to make sure that the **client** is aware of **how the content is structured** on the website. Some of the most important things to be aware of are
+
+- Having different heading sizes in the correct order (H1 -> H2 -> H3 -> H4)
+- No more than one H1 tag
+- Images in the content with set alt tags
+- Enough of inbound and outbound links
+- Set meta descriptions
+- Images shouldn't be too large
+- Set the OG images for the page/post

```

Comment:
Perhaps OG isn't a well known acronym for people who aren't in the loop. Consider explaining that it refers to Open Graph or linking to https://ogp.me/